pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/phimanifest/PhiManifestResponse.java

```diff
@@ -0,0 +1,18 @@
+package org.broadinstitute.dsm.model.phimanifest;
+
+import lombok.Setter;
+
+@Setter
+public class PhiManifestResponse {
```

Comment:
If this is specific to route handling, it should probably be closer to the related route, not in the model.  Or it could move to a static class inside of PhiManifest, and then be returned in a method.

Reply:
Good idea, I moved it